src/screens/Assistant/FadeBetween.tsx

```diff
@@ -21,16 +21,19 @@ type FadeProps = {
   style?: ViewStyle;
   children: [ReactElement, ReactElement];
   duration?: number;
+  isParentAnimating?: boolean;
```

Comment:
Should this just not be optional? Or at the very least defaulted to `false`

Reply:
Sure, can definitely default it to false 👍